odinochka.html

```diff
@@ -55,8 +55,7 @@
     input[name=importfile][value]:not([value=""]) ~ button[name=import] {display:block}
 
     div#groups {
-        display: flex;
-        flex-direction: column;
+        min-height: calc(100vh - 164px);
```

Comment:
The min-height is a good change, but I need flex-direction for controlling the sort order (an option in the menu).